### DISCUSSION TITLE
Runner doesn't pick up job

### DISCUSSION THREAD

**Discussion by michalramus** [actions/runner#2003] [Q&A]
I started job by pushing commit, but no runner picks it up.
https://github.com/Michal123456747/Smart-Home-MGM/actions/runs/2662308877

**Comment by henrygriffiths** [comment]
Hi @Michal123456747,

Unfortunately that action run has expired or has been deleted. From looking at your repo, the latest push to the `master` branch has passing checks. Are you still experiencing this issue?

**Comment by michalramus** [comment]
No, I fixed it. I had typo in configuration script :)
